pkg/apis/kops/cluster.go

```diff
@@ -635,6 +639,8 @@ type ClusterSubnetSpec struct {
 	Type SubnetType `json:"type,omitempty"`
 	// PublicIP to attach to NatGateway
 	PublicIP string `json:"publicIP,omitempty"`
+	// define aws subnet number in decimanl
+	AwsIpv6SubnetNum *int `json:"awsIpv6SubnetNum,omitempty"`
```

Comment:
Do you intend people to write 0-255 instead of 0-FF? I would at least add this to the documentation

Reply:
yes decimal would be good I think and anyway netnum value is in decimal and I'll update the document when this PR is finalized.